aptos-move/move-examples/common_account/sources/common_account.move

```diff
@@ -0,0 +1,269 @@
+/// A framework for sharing a single nesource account across multiple accounts.
+///
+/// This creates a resource account with the ability for other signified accounts the ability to
+/// generate the resource accounts signer. Specifically, the creator maintains the ability to add
+/// and remove new accounts that have access to the resource account signer.
+module common_account::common_account {
+    use std::error;
+    use std::signer;
+
+    use aptos_std::simple_map::{Self, SimpleMap};
+
+    use aptos_framework::account::{Self, SignerCapability};
+
+    /// Missing the common account Management
+    const ENO_MANAGEMENT_RESOURCE_FOUND: u64 = 1;
+    /// Missing CommonAccount
+    const ENO_ACCOUNT_RESOURCE_FOUND: u64 = 2;
+    /// Missing the common account Capability
+    const ENO_CAPABILITY_FOUND: u64 = 3;
+    /// Was not offered a capability
+    const ENO_CAPABILITY_OFFERED: u64 = 4;
+    /// Signer isn't admin
+    const ENOT_ADMIN: u64 = 5;
+    /// Found an address different than expected at the Capability
+    const EUNEXPECTED_PARALLEL_ACCOUNT: u64 = 6;
+
+    /// Contains the signer capability that generates the common account signer.
+    struct CommonAccount has key {
+        signer_cap: SignerCapability,
+    }
+
+    struct Empty has drop, store {}
+
+    /// Contains the framework for managing the account, partiuclarly around adminstration.
+    struct Management has key {
+        admin: address,
+        unclaimed_capabilities: SimpleMap<address, Empty>,
+    }
+
+    /// A revokable capability that is stored on a users account.
+    struct Capability has drop, key {
```

Comment:
This limits that Alice can only join the management group of only one resource account. 
`SmartTable` plz.

Reply:
@gregnazario

Reply:
That's the intent... we can add more complexity if we want, but that also complicates our guides and tutorials.